writers/writer.go

```diff
@@ -76,3 +92,70 @@ func (c *consumer) channelExists(channel string) bool {
 	_, found := c.channels[channel]
 	return found
 }
+
+func (c *consumer) subtopicExists(subtopic string) bool {
+	if _, ok := c.subtopics["*"]; ok {
+		return true
+	}
+
+	_, found := c.subtopics[subtopic]
+	return found
+}
+
+type filterConfig struct {
+	List []string `toml:"filter"`
+}
+
+type channelsConfig struct {
+	Channels filterConfig `toml:"channels"`
+}
+
+func LoadChannelsConfig(chanConfigPath string) map[string]bool {
+	data, err := ioutil.ReadFile(chanConfigPath)
+	if err != nil {
+		log.Print(fmt.Sprintf("Failed to open configuration file: %s", err))
```

Comment:
Use `mainflux/errors` package instead of `"log"`, create an error for this message and return `errors.Wrap(errOpenConfFile, err)`.

Reply:
I have return the error message but I don't know how print the message because at this point the logger (`logger, err := logger.New(os.Stdout, cfg.logLevel)`) isn't already loaded.